helium-lib/src/client.rs

```diff
@@ -651,6 +651,7 @@ pub mod config {
                 let mut req = GatewayInfoStreamReqV1 {
                     signer: self.keypair.public_key().into(),
                     batch_size: 1000,
+                    device_types: vec![],
```

Comment:
if GatewayInfoStreamReqV1 works this would avoid this happening in the fugure
```
         let mut req = GatewayInfoStreamReqV1 {
                    signer: self.keypair.public_key().into(),
                    batch_size: 1000,
                    ..Default::default(),
                };
```